webapp/src/language/en.json

```diff
@@ -72,6 +72,7 @@
       "text17": "We will commit to being held accountable by engaging in check up calls and providing public financial and operation reports on a regular basis.",
       "text18": "We acknowledge that we can be unvoted immediately and won’t be eligible for votes from this DAO ever again if we engage with compromised votes hereafter.",
       "text19": "We will commit to not operating more than one BP with the same ownership."
-    }
```

Comment:
@leisterfrancisco  can you add the spanish translation and include this in english for the other language files